shared/constants/network.ts

```diff
@@ -289,17 +303,22 @@ export const BUILT_IN_NETWORKS = {
     ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.SEPOLIA],
     blockExplorerUrl: `https://${NETWORK_TYPES.SEPOLIA}.etherscan.io`,
   },
-  [NETWORK_TYPES.LINEA_TESTNET]: {
-    networkId: NETWORK_IDS.LINEA_TESTNET,
-    chainId: CHAIN_IDS.LINEA_TESTNET,
-    ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_TESTNET],
+  [NETWORK_TYPES.LINEA_GOERLI]: {
+    networkId: NETWORK_IDS.LINEA_GOERLI,
+    chainId: CHAIN_IDS.LINEA_GOERLI,
+    ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_GOERLI],
     blockExplorerUrl: 'https://explorer.goerli.linea.build',
   },
   [NETWORK_TYPES.MAINNET]: {
     networkId: NETWORK_IDS.MAINNET,
     chainId: CHAIN_IDS.MAINNET,
     blockExplorerUrl: `https://etherscan.io`,
   },
+  [NETWORK_TYPES.LINEA_MAINNET]: {
+    networkId: NETWORK_IDS.LINEA_MAINNET,
+    chainId: CHAIN_IDS.LINEA_MAINNET,
+    blockExplorerUrl: 'https://lineascan.build',
```

Comment:
This site does not seem to be working

Reply:
Yes this url is not working for the moment but will be the one for linea mainnet :)